test/system/ClientAppFixture.h

```diff
@@ -24,14 +24,15 @@ class ClientAppFixture : public ClientApp {
 
     int EffectsProcessingThreads() const override;
 protected:
-    bool                 m_game_started;   ///< Is server started the game?
-    std::set<int>        m_ai_empires;     ///< Ids of AI empires in game.
-    std::set<int>        m_ai_waiting;     ///< Ids of AI empires not yet send orders.
-    bool                 m_turn_done;      ///< Is server processed turn?
-    bool                 m_save_completed; ///< Is server saved game?
-    boost::uuids::uuid   m_cookie;         ///< Cookie from server login.
-    bool                 m_lobby_updated;  ///< Did player get updated lobby.
-    MultiplayerLobbyData m_lobby_data;     ///< Lobby data.
+    bool                 m_game_started;    ///< Is server started the game?
+    std::set<int>        m_ai_empires;      ///< Ids of AI empires in game.
+    std::set<int>        m_ai_waiting;      ///< Ids of AI empires not yet send orders.
+    bool                 m_turn_done;       ///< Is server processed turn?
+    bool                 m_save_completed;  ///< Is server saved game?
+    boost::uuids::uuid   m_cookie;          ///< Cookie from server login.
+    bool                 m_lobby_updated;   ///< Did player get updated lobby.
+    MultiplayerLobbyData m_lobby_data;      ///< Lobby data.
+    bool                 m_use_compression; ///< Compression use request flag from the server.
```

Comment:
What does this track? If the server has requested that this client send back compressed messages?

Reply:
Variable removed in https://github.com/freeorion/freeorion/pull/3854/commits/e937091c4540e765549e9147757117dfbe3f4bcf.